### DISCUSSION TITLE
Remarks on automation/scripts/etc/etc

### DISCUSSION THREAD

**Discussion by MAFL74** [home-assistant/frontend#19358] [Automations / scripts / scenes]
What about having 'remarks' available when creating big/advanced automations to make it easier to keep track of things? 
See example below...

![Image1](https://github.com/home-assistant/frontend/assets/36964445/2484d30e-0405-4cfd-80f8-166fe2cdfd5b)

**Comment by karwosts** [comment]
Couldn't you use the rename/alias option for each item for a similar effect? 

If that was my automation, I think it would make sense to just rename each element to exactly the strings you have provided as "remarks".

**Reply by MAFL74** [reply]
That might be a posibility, but I think it would be nicer and better overview to have the remarks easily available for editing/changing on the same page.
